src/components/common/FileUploadComponent.tsx

```diff
@@ -366,6 +366,7 @@ export function FileUploadComponent(props: {
   maxFiles?: number;
   id?: string;
   fileType: string;
+  pdfPageLimit: number;
```

Comment:
Maybe it is a good idea to have a comment here that `0` means unlimited pages. Because I see we use the component in couple of places and would be nice in the future if we want to add to another place to know that we should set this property to 0 for unlimited.

Reply:
Would also be nice to let the user know it somehow

Reply:
Thanks both 🙂

@jekabs-karklins I think this exists at the moment, unless you meant something more?

![image](https://user-images.githubusercontent.com/14293717/173575426-3d0f029b-1b25-49e8-bf87-403622f65aa6.png)

![image](https://user-images.githubusercontent.com/14293717/173575539-76700457-f9e3-43ca-bf70-a1ab8f6ffe49.png)